ProcessLib/SmallDeformation/CreateLocalAssemblers.h

```diff
@@ -30,7 +30,6 @@ template <unsigned GlobalDim, int DisplacementDim,
 void createLocalAssemblers(
     NumLib::LocalToGlobalIndexMap const& dof_table,
     std::vector<MeshLib::Element*> const& mesh_elements,
-    unsigned const integration_order,
```

Comment:
The integration order is now contained in `extra_ctor_args`. That way one does not need to declare additional arguments in utility code, like `createLocalAssemblers()`, anymore.